qdrant-landing/content/documentation/frameworks/openlit.md

```diff
@@ -0,0 +1,53 @@
+---
+title: OpenLIT
+weight: 3100
+---
+
+# OpenLIT
+
+[OpenLIT](https://github.com/openlit/openlit) is an OpenTelemetry-native LLM Application Observability tool and includes OpenTelemetry auto-instrumention for Qdrant with just a single line of code helping you ensure your applications are monitored seamlessly, providing critical insights to improve performance, operations and reliability. 
+
+For more information on how to use OpenLIT, see the [OpenLIT docs](https://docs.openlit.io/).
+
+![OpenLIT Connections Banner](https://github.com/openlit/.github/blob/main/profile/assets/github-readme-connections-banner.png?raw=true)
+
+This page assumes you're using `qdrant-client` version 1.7.3 or above.
+
+## Usage
+
+### Step 1: Install OpenLIT
+
+Open your command line or terminal and run:
+
+```bash
+pip install openlit
+```
+
+### Step 2: Initialize OpenLIT in your Application
+Integrating OpenLIT into LLM applications is straightforward. Start monitoring for your LLM Application with just **two lines of code**: 
+
+```python
+import openlit
+
+openlit.init()
+```
+
+To forward telemetry data to an HTTP OTLP endpoint, such as the OpenTelemetry Collector, set the `otlp_endpoint` parameter with the desired endpoint. Alternatively, you can configure the endpoint by setting the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable as recommended in the OpenTelemetry documentation.
+
+> 💡 Info: If you don't provide `otlp_endpoint` function argument or set the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable, OpenLIT directs the trace directly to your console, which can be useful during development.
+To send telemetry to OpenTelemetry backends requiring authentication, set the `otlp_headers` parameter with its desired value. Alternatively, you can configure the endpoint by setting the `OTEL_EXPORTER_OTLP_HEADERS` environment variable as recommended in the OpenTelemetry documentation.
```

Comment:
You can use https://github.com/qdrant/landing_page/blob/e3117ff503ca51c5a1b3a768bf56da9b6e685d8a/qdrant-landing/content/documentation/quick-start.md?plain=1#L131 for notices.